### ISSUE TITLE
Headers with variables don't work

### ISSUE THREAD

**Issue by adam-fowler** [vknabel/vscode-swiftformat#18]
I have a comment header defined in my `.swiftformat` file which includes the `{created}` variable. The swift format extension fails to run while this is included. It returns with the error
```
Failed to apply {created} template in file header as file info is unavailable
```
Add this to your `.swiftformat` to replicate
```
--header "// Created {created}"
```

**Comment by vknabel** [comment]
Thanks for the report!
Didn't know about the feature!

I'll have a look on the swiftformat docs and work on a fix soon

**Comment by vknabel** [comment]
Fix has been released in 1.3.8
